test/remote/db_tests.js

```diff
@@ -139,11 +128,10 @@ describe('remote db', function() {
     () => {
       let tokenId
 
-      // Simulate a cache miss in redis
-      redisGetSpy.returns(P.resolve(null))
-
-      // Fetch all sessions for the account
-      return db.sessions(account.uid)
+      // Ensure redis is empty for the uid
+      return redis.delAsync(account.uid)
```

Comment:
Noticed `redis.delAsync` was used a couple times, should it be put into a `beforeEach`?

Reply:
Good shout, will do.